opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/RuntimeEnvironment.java

```diff
@@ -110,6 +111,8 @@
 
     private Statistics statistics = new Statistics();
 
+    public IncludeFiles includeFiles;
```

Comment:
public? please no

Reply:
what would be the alternative ?

Reply:
I know that exposing it with getter is in a sense the same, but keep it please consistent with the rest of the file. This class is the most used in OpenGrok so rather make it clean.

Reply:
okay, used getter.